Tools/clinic/clinic.py

```diff
@@ -5335,8 +5375,15 @@ def at_critical_section(self, *args: str) -> None:
         self.critical_section = True
 
     def at_getter(self) -> None:
+        if self.kind in (GETTER, SETTER):
+            fail("only one of @getter or @setter can be used")
```

Comment:
From the user's perspective, I think these are actually two distinct errors, so it makes sense to give two different error messages:

```suggestion
        if self.kind is GETTER:
            fail("Cannot apply @getter twice to the same function!")
        if self.kind is SETTER:
            fail("Cannot apply both @getter and @setter to the same function!")
```

Reply:
It also looks like this failure path might not have any test coverage; could you add a test for it? :)